### ISSUE TITLE
TypeError: Return value of Doctrine\DBAL\Connection::getWrappedConnection() must implement interface Doctrine\DBAL\Driver\Connection, instance of PDO returned

### ISSUE THREAD

**Issue by mvorisek** [doctrine/dbal#4609]
### BC Break Report

|    Q        |   A
|------------ | ------
| BC Break    | yes
| Version     | 2.13.0

#### Summary

connection is created like:

```
$dbalConnection = DriverManager::getConnection([
    'pdo' => $pdo,
]);
```

and then used in:
```
$dbalConnection->executeQuery('DROP TABLE IF EXISTS employee');
```

which started throwing:

```
TypeError: Return value of Doctrine\DBAL\Connection::getWrappedConnection() must implement interface Doctrine\DBAL\Driver\Connection, instance of PDO returned

...\vendor\doctrine\dbal\lib\Doctrine\DBAL\Connection.php:1947
...\vendor\doctrine\dbal\lib\Doctrine\DBAL\Connection.php:1277
...\vendor\atk4\dsql\tests\WithDb\TransactionTest.php:33
```

#### Previous behaviour

no error

#### Current behavior

see log above

#### How to reproduce

see code above

**Comment by beberlei** [comment]
Fixed by #4590

**Comment by mvorisek** [comment]
Thanks, I see, fixed with 2.13.1 rel. released 3 hours ago.
